hazelcast/src/main/java/com/hazelcast/client/impl/protocol/task/transaction/XATransactionRollbackMessageTask.java

```diff
@@ -74,7 +74,7 @@ public String getDistributedObjectName() {
 
     @Override
     public String getMethodName() {
-        return null;
+        return "transactionRollback";
```

Comment:
The method name for this is just rollback. See XAResource.rollback.

Reply:
changed.